runtime/src/snapshot_utils.rs

```diff
@@ -2965,6 +2958,21 @@ pub fn purge_old_bank_snapshots(
     do_purge(bank_snapshots);
 }
 
+/// Purges all `bank_snapshots`
+///
+/// Does not exit early if there is an error while purging a bank snapshot.
+fn _purge_bank_snapshots<'a>(bank_snapshots: impl IntoIterator<Item = &'a BankSnapshotInfo>) {
```

Comment:
why the prefixed `_` in the function name? I think the name already implies it purges them all which distinguishes it from `purge_old_bank_snapshots`

Reply:
Please update the PR description/name accordingly so the merge commit message isn't misleading 😄